README.rst

```diff
@@ -146,13 +146,13 @@ Docker
 Support for running Contextualise within `Docker <https://www.docker.com/>`_ is still in development. To run it from the
 root of a local clone of the source:
 
-1. Create a ``settings.ini`` file in the root, for example::
+1. Copy ``settings-docker-sample.ini`` to ``settings.ini`` file in the root and replace your email settings. For example::
```

Comment:
Could there be a single `settings-sample.ini` with the compose postgres values, but with placeholder email settings? I came here to test drive this as well, and would prefer to look around (I trust it sends email).

If so, you could update the `Dockerfile` to `RUN mv settings-sample.ini settings.ini` and this should run out of the box.